code-review/infrateam_prs.md

```diff
@@ -0,0 +1,13 @@
+# Code Review
+
+## Pull Request Practices in the Infrastructure Team
+
+The Infrastructure Team uses the following practices with the goal of good communication around the status and needed actions for a PR. 
+
+* When reviewing a PR, change the PR assignee in GitHub to yourself. This allows others to be aware that the PR is receiving attention.  This does not mean that another developer cannot also review and comment on the PR at any point in its review, even after merging. Review by more than one developer is not wasted effort.
```

Comment:
It might be duplicative of what's already here, but I wonder if another bullet immediately after this one---or maybe a sub-bullet?---might be a good fit here:

```markdown
* Before merging a PR, make sure all PR assignees have submitted a review already
```

Or something like that?

Reply:
Thanks, @mjgiarlo. I'll add that as a third bullet, if that works OK for what you wanted to get across. I see the first two bullets as applying to the "reviewer" role and the new (third) one as the "merger" role.